packages/base-ui/src/services/confirm/desktop-confirm.service.ts

```diff
@@ -0,0 +1,24 @@
+import { toDisposable } from '@univerjs/core';
+import { IDisposable } from '@wendellhu/redi';
+import { Subject } from 'rxjs';
+
+import { type IConfirmPartMethodOptions } from '../../views/components/confirm-part/interface';
+import { IConfirmService } from './confirm.service';
+
+export class DesktopConfirmService implements IConfirmService {
```

Comment:
The service seems to be designed on an assumption that there is only one confirm dialog at a time. The assumption may need to be re-evaluated.